src/main/java/HelloWebDriver.java

```diff
@@ -0,0 +1,23 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+
+    public class HelloWebDriver {
+
+        public static void main(String[] args)  {
+
+            WebDriver driver = new ChromeDriver();
+            driver.get("http://google.com");
+            driver.quit();
+
+            System.setProperty("webdriver.edge.driver", "C:\\Webdriver\\msedgedriver.exe");
```

Comment:
All three drivers should be placed into the project. I don't have "C:\Webdriver\msedgedriver.exe" path on my laptop, so when I'll try run tests - they will fail.

Reply:
Thank you for pointing.
Please see changes in the following commit.